### ISSUE TITLE
site support request for americastestkitchen.com

### ISSUE THREAD

**Issue by Transcodes** [rg3/youtube-dl#10764]
- [x] I've **verified** and **I assure** that I'm running youtube-dl **2016.09.24**
### Before submitting an _issue_ make sure you have:
- [x] At least skimmed through [README](https://github.com/rg3/youtube-dl/blob/master/README.md) and **most notably** [FAQ]
- [x] [Searched](https://github.com/rg3/youtube-dl/search?type=Issues) the bugtracker for similar issues including closed ones
### What is the purpose of your _issue_?
- [ ] Bug report (encountered problems with youtube-dl)
- [x] Site support request (request for adding support for a new site)
- [ ] Feature request (request for a new functionality)
- [ ] Question
- [] Other
### If the purpose of this _issue_ is a  _site support request_ or you are not completely sure provide the full verbose output as follows:

Add `-v` flag to **your command line** you run youtube-dl with, copy the **whole** output and insert it here. It should look similar to one below (replace it with **your** log inserted between triple ```):

```
$ youtube-dl -v https://www.americastestkitchen.com/episode/115-beef-stew
[debug] System config: []
[debug] User config: []
[debug] Command-line args: ['-v', 'https://www.americastestkitchen.com/episode/115-beef-stew']
[debug] Encodings: locale cp1252, fs mbcs, out cp437, pref cp1252
[debug] youtube-dl version 2016.09.24
[debug] Python version 3.4.4 - Windows-10-10.0.10586
[debug] exe versions: none
[debug] Proxy map: {}
[generic] 115-beef-stew: Requesting header
WARNING: Falling back on generic information extractor.
[generic] 115-beef-stew: Downloading webpage
[generic] 115-beef-stew: Extracting information
ERROR: Unsupported URL: https://www.americastestkitchen.com/episode/115-beef-stew
Traceback (most recent call last):
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\rg3\tmpc7h_sle1\build\youtube_dl\YoutubeDL.py", line 694, in extract_info
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\rg3\tmpc7h_sle1\build\youtube_dl\extractor\common.py", line 355, in extract
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\rg3\tmpc7h_sle1\build\youtube_dl\extractor\generic.py", line 2424, in _real_extract
youtube_dl.utils.UnsupportedError: Unsupported URL: https://www.americastestkitchen.com/episode/115-beef-stew

<end of log> 
```
### If the purpose of this _issue_ is a _site support request_ please provide all kinds of example URLs support for which should be included
- Single video: https://www.americastestkitchen.com/episode/511-revolutionizing-classic-fruit-desserts
- Single video: https://www.americastestkitchen.com/episode/110-tomato-sauces-for-pasta
- Playlists : 

All https://www.americastestkitchen.com/episodes/browse

Season wise
https://www.americastestkitchen.com/episodes/16
.
.
.
https://www.americastestkitchen.com/episodes/1
### Description of your _issue_, suggested solution and other information

Video plays directly in the browser
no account needed 
No Geo-restriction applied
Subtitles available..please check if they can be retrieved as well

please comment If you need any more clarification
your help is greatly appreciated
thank you very much

**Comment by nbppp2** [comment]
So I have been looking at this recently & noticed that the sister site cookscountry.com works just fine. In short it appears that as the general extractor tries to parse the site, there is some malformed xml in there. I printed out the xml that is failing to be parsed & tried to parse it with [https://www.freeformatter.com/xml-formatter.html](https://www.freeformatter.com/xml-formatter.html). I get 

`Element type "path" must be followed by either attribute specifications ">" or "/>".`

Which to me doesn't give really any good information other than the xml is malformed.

I'll take a further look to see if I can figure out what is malformed on the episode pages when I have some free time.

**Comment by nbppp2** [comment]
Ok, so I have relinquished the possibility that this could be handled by the generic extractor like it's sister site. 

After working to implement an extractor I have seem to run into a few issues, but the ATK website seems have the same setup as #9200. 

For example  [https://www.americastestkitchen.com/episode/548-summer-dinner-party](https://www.americastestkitchen.com/episode/548-summer-dinner-party)  can be downloaded by doing youtube-dl -v "kaltura:1445801:1_5g5zua6e".

Sadly I tried to download a video from the page referenced in the other extractor & got an error. Sadly it is the same error I ran into when writing my extractor. However progress does seem to be being made.
